packages/vscode-mdx/package.json

```diff
@@ -46,15 +46,14 @@
   },
   "devDependencies": {
     "@types/node": "^20.0.0",
-    "@types/vscode": "^1.0.0",
-    "@volar/language-server": "~1.10.0",
-    "@volar/vscode": "~1.10.0",
+    "@types/vscode": "^1.82.0",
+    "@volar/language-server": "~1.11.1",
+    "@volar/vscode": "~1.11.1",
```

Comment:
I'll defer to @remcohaszing on this, elsewhere in unified, we don't update ranges unless the new version is outside the allowed range.
These appear to already be within allowed range.

Reply:
Version restrictions have been narrowed for consistency

Reply:
Please note that I left "@types/vscode": "^1.82.0" as this is the de facto minimum requirement.